src/explore-education-statistics-common/src/modules/methodology/components/MethodologyHelpAndSupportSection.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import ContactUsSection from '@common/modules/find-statistics/components/ContactUsSection';
+import { PublicationSummary } from '@common/services/publicationService';
+import { MethodologyPublication } from '@common/services/methodologyService';
+
+interface Props {
+  owningPublication: PublicationSummary | MethodologyPublication;
+}
+
+const MethodologyHelpAndSupportSection: FC<Props> = ({ owningPublication }) => {
```

Comment:
Few things:

## Usage of `FC` type

Don't use `FC` unless your component actually has `children`. `FC` includes a `children`  prop by default and this doesn't make sense if you don't actually render any children.

We also typically just define the children in the props interface itself anyway so that readers can know exactly what the props are at a glance. Using `FC` could lead to some minor confusion about where the `children` are coming from.

This said, we sometimes use `FC` deliberately in cases where we have a simple wrapping component that renders some children.

## Component declaration syntax

You'll probably see that in a lot of the codebase, we've written components with arrow functions. Recently, we've kind of switched to defining exported components and functions with normal function syntax. 

This is mostly so that we can condense the declaration and `export default` into a single line:

```ts
export default function YourFunction() {
}
```

This is a little minor, but something I'd probably encourage.

Reply:
Changed.